components/Description-Scott/AddToCart/SelectQuantity.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import DropdownButton from 'react-bootstrap/DropdownButton';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+const SelectQuantity = ({ sku, setQuantitySelected }) => {
+  const [currentQuantity, setCurrentQuantity] = useState(1);
+  const [quantityAvailable, setQuantityAvailable] = useState(0);
+
+  const checkQuantity = () => {
+    if (sku) {
+      setQuantityAvailable(sku.quantity);
+      setCurrentQuantity(1);
+    }
+  };
+
+  const clickHandler = (quantity) => {
+    setQuantitySelected(quantity);
+    setCurrentQuantity(quantity);
+  };
+
+  useEffect(() => {
+    checkQuantity();
+  }, [sku]);
```

Comment:
Does this work since it is state being passed down? Pretty cool if so!

Reply:
yep!